src/test/scala/genwasym/TestSyntax.scala

```diff
@@ -0,0 +1,17 @@
+package gensym.wasm
+
+import gensym.wasm.parser.Parser
+import org.scalatest.FunSuite
+
+class TestSyntax extends FunSuite {
```

Comment:
Then it seems we don't need this test (since we have TestScriptRun)?

Reply:
My first concern is that with this test, syntax development (related to ANTLR files) and semantic development could become separated. But now I think we can delete them, since there is already an almost finished ANTLR grammar file.